tests/parser/features/test_assert.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from tests.setup_transaction_tests import chain as s, tester as t, ethereum_utils as u, check_gas, \
+    get_contract_with_gas_estimation, get_contract
+
+def test_assert_refund(t):
+    code = """
+@public
+def foo():
+    assert 1 == 2
+"""
+    c = get_contract_with_gas_estimation(code)
+    pre_balance = t.s.head_state.get_balance(t.a0)
+    with pytest.raises(t.TransactionFailed):
+        c.foo(startgas=1000000, gasprice=10)
+    post_balance = t.s.head_state.get_balance(t.a0)
+    # Checks for gas refund from revert
+    assert pre_balance < post_balance + 100000
```

Comment:
Missing a zero. Can you use `1e6` instead?

Reply:
It's supposed to be 100,000 to account for the initial gas used in the function call before the revert, but I'll change it to `10**5`

Reply:
But you supply 1e6... I guess it's not the clearest why those two numbers work. Can you add a comment like "this makes sense because about X gas will be used and 1e6 - X is around 1e5 left over". Something like that.